### PR TITLE
Availability Page

### PR DESCRIPTION
**Pull request by meagancheese**
_Edit by acmcarther@_
Screenshot 2020-06-24 at 5 09 21 PM

### CODE REVIEW

File: src/main/webapp/availability.html [hunk 1]
```diff
@@ -0,0 +1,403 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Availability</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="availability.css">
```
Comment (apluscs) [inline src/main/webapp/availability.html:9]: Just a reminder to rebase onto master and remove the #content from availability.css

File: src/main/webapp/availability.html [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+          aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+          <div class="navbar-nav">
+            <a class="nav-item nav-link" href="index.html">Home</a>
+            <!--TODO: Put in links to other pages. Put on all pages-->
+            <a class="nav-item nav-link" href="#">Profile</a>
+            <a class="nav-item nav-link" href="#">Events</a>
+            <a class="nav-item nav-link active" href="#">Availability<span class="sr-only">(current)</span></a>
```
Comment (apluscs) [inline src/main/webapp/availability.html]: The span doesn't do anything. Best to leave it out.

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -29,7 +29,7 @@
             <!--TODO: Put in links to other pages. Put on all pages-->
             <a class="nav-item nav-link" href="#">Profile</a>
             <a class="nav-item nav-link" href="#">Events</a>
-            <a class="nav-item nav-link" href="#">Availability</a>
+            <a class="nav-item nav-link" href="availability.html">Availability</a>
```
Comment (apluscs) [inline src/main/webapp/index.html:33]: How are we doing this navbar incorporation? l thought we would leave all the links as "#" for now and then create a new branch after all the HTMLs are added into master, then add all links to all of their navbars. lf we're sticking with this way, can you remove the `href="availability.html"` here for consistency? lt bugs me how we have  `href="availability.html"` but not the same for Profile.

File: src/main/webapp/availability.html [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+          </div>
+        </div>
+      </nav>
+      <br>
+      <h1 class="text-center">Availability</h1>
+      <h5 class="text-center lead">Update your availability for this week below:</h5>
+      <br>
+      <!--TODO: Include Timezone consideration; Check inputs for correctness: end is after start, at least an hour long;
+         Add pagination; Figure out how to update the dates; store and show inputted data; Allow for more than one
+         time range per day (have a plus button)-->
```
Comment (apluscs) [inline src/main/webapp/availability.html:46]: Definitely want to generate these days and dropdowns dynamically through some calculation in Java servlets or something. Include in the TODOs?

File: src/main/webapp/availability.css [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+.center-with-form {
+  margin-top: 5px;
+  margin-bottom: 0;
+}
+
+.end-time {
+  margin-left: 8px;
+}
+
+.space-above {
```
Comment (apluscs) [inline src/main/webapp/availability.css]: Regarding `.center-with-form`, `.end-time`, and `.space-above` : l don't know if we should name classes based on what they do or what they are applied to. l personally like the latter method, but am unsure.
Thoughts @jalexanderqed @acmcarther ?

**Review by apluscs — CHANGES_REQUESTED** [review]

**Comment by acmcarther** [comment]
Include screenshot in PR description?

File: src/main/webapp/availability.css (hunk 5 above)
Comment (acmcarther) [reply src/main/webapp/availability.css]: describing the class by what it does internally is strictly worse than describing it by what it is.

What it does: "space-above"
What it is: "schedule-card" (just made that up)

**Comment by meagancheese** [comment]
The screenshot is one of the files, you need to click on Rich Div Format or something to see it.

File: src/main/webapp/availability.html (hunk 1 above)
Comment (meagancheese) [reply src/main/webapp/availability.html:9]: Done

File: src/main/webapp/availability.html (hunk 2 above)
Comment (meagancheese) [reply src/main/webapp/availability.html]: Done

File: src/main/webapp/index.html (hunk 3 above)
Comment (meagancheese) [reply src/main/webapp/index.html:33]: Now that I pulled from master, I updated the navbars to the current state of this branch (links for all but scheduling and logout).

File: src/main/webapp/availability.html (hunk 4 above)
Comment (meagancheese) [reply src/main/webapp/availability.html:46]: Done

**Comment by meagancheese** [comment]
Screenshot 2020-06-24 at 5 09 21 PM

File: src/main/webapp/availability.css (hunk 5 above)
Comment (meagancheese) [reply src/main/webapp/availability.css]: Done

File: src/main/webapp/style.css [hunk 6]
```diff
@@ -1,9 +1,3 @@
-#content {
```
Comment (acmcarther) [inline src/main/webapp/style.css:1]: Is this change deliberate?

**Review by acmcarther — APPROVED** [review]
I haven't really scrutinized the HTML much. LMK if you'd like a detailed look at that.

File: src/main/webapp/style.css (hunk 6 above)
Comment (meagancheese) [reply src/main/webapp/style.css:1]: Yes, the content id is being handled in the global.css file now.

**Review by apluscs — APPROVED** [review]
